tools/font-subset/test.py

```diff
@@ -77,9 +80,21 @@ def RunCmd(cmd, codepoints, fail=False):
   return p.returncode
 
 
+def TestZip():
+  with ZipFile(FONT_SUBSET_ZIP, 'r') as zip:
+    files = zip.namelist()
+    if 'font-subset%s' % EXE not in files:
+      print('expected %s to contain font-subset%s' % (files, EXE))
```

Comment:
This will fail if it got inserted into the zip as `Users/me/src/flutter/engine/src/flutter/tools/font-subset` instead of just `font-subset`.